### PR TITLE
Add and clarify comments and variable names, processing steps

### PR DESCRIPTION
**Pull request by bpbond**

### CODE REVIEW

File: R_scripts/licorplot.R [hunk 1]
```diff
@@ -10,44 +10,57 @@ library(readr)
...
+plotDat <- read_csv("../design/plots.csv", col_types = "ccccddi")
+
+cat("Joining datasets and calculating...\n")
+
+# Merge these three datasets together based on collar number and plot name
+licorDat %>% 
+  left_join(collarDat, by = "Collar") %>% 
   rename(Origin_Plot = Plot) %>%
-  select(-Site)
-dat <- left_join(dat, plots, by = c("Origin_Plot" = "Plot")) %>%
```
Comment (bpbond) [inline R_scripts/licorplot.R]: `dat` is not informative–could be anything.

**Comment by stephpenn1** [comment]
So, if I want to make revisions to your code do I download it to my local?

**Comment by bpbond** [comment]
So if you want to make changes to this branch you can do so, and push changes to it.

When you say "my own version" do you mean the `steph` branch?

**Comment by stephpenn1** [comment]
Yes, I have changes on my local that haven't been pushed to the `steph` branch yet

**Comment by bpbond** [comment]
I'd push to `steph` and open a pull request to `master`. Then we can reconcile everything.

**Comment by stephpenn1** [comment]
Oh! Do I switch to the master branch to make changes? And then when we merge the branches, resolve conflicts?

**Comment by bpbond** [comment]
Let's start by you committing to `steph` and open a PR to `master`, and go from there.

**Comment by bpbond** [comment]
We should also talk about the wisdom of keeping a permanently-open `steph` branch versus opening and closing branches for specific tasks.

**Comment by bpbond** [comment]
I think I'll close this and re-do it against your new code.

**Comment by stephpenn1** [comment]
Okay, I do like your comment additions and revisions, though!

**Pull request closed without merging**